md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/BatchAPI.java

```diff
@@ -184,10 +177,10 @@ RestWrapper update(@ModelAttribute("batch")
             }
 
             batchDAO.update(jpaBatch);
-            // Batch batches = s.selectOne("call_procedures.UpdateBatch", batch);
             restWrapper = new RestWrapper(batch, RestWrapper.OK);
-            LOGGER.info("Record with ID:" + batch.getBatchId() + " updated in Batch by User:" + principal.getName() + batch);
+            LOGGER.info(RECORDWITHID + batch.getBatchId() + " updated in Batch by User:" + principal.getName() + batch);
         } catch (Exception e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1166)